### ISSUE TITLE
Add additional run parameters to groups in composition files

### ISSUE THREAD

**Issue by KatarinaKeti** [testground/testground#1343]
## What defines this endeavour to be complete?

<!--Create a check list of what you expect to see as part of delivering on the ask--

- [ ] Item 1
- [ ] Item 2
- [ ] ...

**Comment by brdji** [comment]
>What defines this endevour to be complete?

Add a example usage of additional run parameters (ie. `build_config` override in groups) once #1338 is merged and closed.

**Comment by laurentsenta** [comment]
Could you share more info about this issue?
What problem is this solving?

It looks like a duplicate of https://github.com/testground/testground/issues/1337
but we're also talking about "run parameters" (1337 is concerned with build parameters).

**Comment by laurentsenta** [comment]
@KatarinaKeti Is there a link to the documentation and examples that were added?

**Comment by brdji** [comment]
I've added a composition file that uses various build options that were added:
https://github.com/testground/learning-example-tg/blob/b4c0dee96718b591060cfbb3331f84b9701a1c1f/compositions/tg-build-options.toml
